estate_account/models/estate_property.py

```diff
@@ -0,0 +1,27 @@
+from odoo import Command, models
+
+
+class EstateProperty(models.Model):
+    _inherit = "estate.property"
+
+    def sold_property(self):
+        for estate in self:
+            self.env['account.move'].create(
```

Comment:
For the sake of the tutorial, this is good. But for your knowledge and to keep in mind for your future work, we try to NOT use ORM methods in a loop (create, write, unlink and mainly search) as this kills performances when dealing with huge data